browser-test/test.grpc-fallback.ts

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright 2019 Google LLC
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are
+ * met:
+ *
+ *     * Redistributions of source code must retain the above copyright
+ * notice, this list of conditions and the following disclaimer.
```

Comment:
I think this is a BSD license? this seems a bit weird with the rest of the code base being Apache?

Or it's at least weird that we're mixing and matching Apache and BSD in this repo?

Reply:
Unfortunately, this particular package is BSD licensed (it's an outlier in our world). I have no idea why - but I'm guessing we cannot just relicense it to Apache given its age and some history of external contributions. Since BSD is an approved license, we're OK with it, I'm just trying to keep things consistent here.